app/src/test/java/com/hifeful/mealmania/presentation/MealManiaViewModelTest.kt

```diff
@@ -0,0 +1,121 @@
+package com.hifeful.mealmania.presentation
+
+import android.net.Uri
+import android.support.test.runner.AndroidJUnit4
+import com.hifeful.mealmania.presentation.util.DeeplinkResult
+import org.junit.Test
+import org.junit.runner.RunWith
+
+
+@RunWith(AndroidJUnit4::class)
+class MealManiaViewModelTest {
+    private val viewModel = MealManiaViewModel()
+
+    @Test
+    fun `Given mymeals page deeplink, When handleDeeplink is called, Then ShowMyMealsPage DeeplinkResult should be returned`() {
+        // Given
+        val uri = Uri.parse("mealmania://mymeals")
+        
+        // When
+        val actual = viewModel.handleDeeplink(uri)
+
+        // Then
+        assert(actual is DeeplinkResult.ShowMyMealsPage)
```

Comment:
I know the handy API is provided by the library `org.assertj.core.api.Assertions`:
isEqualTo, isNull, isEmpty, etc.

Reply:
Thank you for the suggestion! 
Usually, I'm using **Google's Truth** library for **assertion** purposes. But I checked and used your library and it's handy to use.